de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/purchase/inout/report_details_v2.jrxml

```diff
@@ -245,7 +245,7 @@
 			<textField isStretchWithOverflow="true" pattern="###0" isBlankWhenNull="true">
 				<reportElement key="textField-24" mode="Transparent" x="255" y="0" width="35" height="12" forecolor="#000000" backcolor="#FFFFFF" uuid="097beb53-5e78-4dd1-9c25-efb41d796b05">
 					<property name="com.jaspersoft.studio.unit.width" value="pixel"/>
-					<printWhenExpression><![CDATA[new Boolean($P{displayhu}.equals( "Y" ))]]></printWhenExpression>
+					<printWhenExpression><![CDATA[new Boolean($P{displayhu}.equals( "Y" ) && !$F{huname}.isEmpty() && $F{huname} != null)]]></printWhenExpression>
```

Comment:
hi @metas-lc 
i think evaluating in this order doesn't make much sense :grin: 
*If* `$F{huname} != null`, then `!$F{huname}.isEmpty()` will already have thrown a NullpointerException :nerd_face: